accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/configreader/YamlMacroElEvaluator.java

```diff
@@ -99,7 +100,8 @@ public ElFunctionMapper() {
                         StringUtils.class.getMethod("split", new Class<?>[] { String.class, String.class }),
                         StringUtils.class.getMethod("join", new Class<?>[] { Object[].class, String.class }),
                         ArrayUtils.class.getMethod("subarray", new Class<?>[] { Object[].class, int.class, int.class }),
-
+                        YamlMacroElEvaluator.class.getMethod("containsElement", new Class<?>[] { List.class, String.class }),

```

Comment:
I would rather go for containsItems(), compare with https://stackoverflow.com/questions/31427117/difference-between-element-item-and-entry-when-it-comes-to-programming

Reply:
yes, that's better +1